app/models/item.rb

```diff
@@ -10,4 +11,12 @@ def current_price
     price = (unit_price * 0.01).round(3)
     "$#{price}"
   end
+
+  def self.enabled_items
+    where('status = 1')
+  end
+
+  def self.disabled_items
+    where('status = 0')
+  end
```

Comment:
This is a more canonical way to do a where clause in Rails
```suggestion
  def self.enabled_items
    where(status: 1)
  end

  def self.disabled_items
    where(status: 0)
  end
```